### PR TITLE
add grid onCreateIsRecordDraggable callback

### PR DESCRIPTION
**Pull request by nebojsa-peric**
provide an option to mark grid rows as non draggable

### CODE REVIEW

**Comment by nebojsa-peric** [comment]
fixes #997

File: packages/cx/src/widgets/grid/Grid.js [hunk 1]
```diff
@@ -1303,6 +1306,13 @@ class GridComponent extends VDOM.Component {
             mod["non-selectable"] = !selectable;
          }
 
+         let draggable =
+            dragSource &&
+            (!row.dragHandles || row.dragHandles.length == 0) &&
+            (!isRecordDraggable || isRecordDraggable(record.data));
+         mod["draggable"] = draggable;
+         mod["non-draggable"] = !draggable;
+
```
Comment (mstijak) [inline packages/cx/src/widgets/grid/Grid.js:1315]: I think this part can be removed as draggability does not influence the visual appearance of the row in any way, but I might be wrong.

**Review by mstijak — APPROVED** [review]
Looks good!

File: packages/cx/src/widgets/grid/Grid.js (hunk 1 above)
Comment (nebojsa-peric) [reply packages/cx/src/widgets/grid/Grid.js:1315]: `draggable` mod was already there, I've just added `non-draggable` to make it equivalent to the selection mods. My idea was that non-draggable records could be visually differentiated from the the ones that are draggable, but not sure whether it's a valid use case.

**Pull request merged**